### PR TITLE
Fix : When the rainfall data is spatially desagregated, a temporary n…

### PR DESCRIPTION
**Pull request by maximejay**
…ew raster geotiff is generated. We guess the no data value using the input raster. However, if nodata value was set toi None type, the setter "_setNoDataValue" crash. Indeed arguments must be a double type only. Thus We know check if the guessed nodata value is an instance of float, else we set nodata value to -99.0

### CODE REVIEW

File: smash/tools/raster_handler.py
Comment (inoelloc) [inline smash/tools/raster_handler.py:1]: Possible de formatter les mofications ? `black smash/tools/raster_handler.py`

**Review by inoelloc — COMMENTED** [review]
Ok pour moi, pas de modification de code. Est ce que tu peux rajouter dans la release note le fix stp

File: smash/tools/raster_handler.py
Comment (maximejay) [reply smash/tools/raster_handler.py:1]: Oui c'est bon :)

**Comment by maximejay** [comment]
Salut,
C'est bon j'ai pushé et backé et updaté
:)

**Review by inoelloc — APPROVED** [review]

**Pull request merged**